File: C#/Ejercicio1.cs
using System;
public class Persona
{
    public string Nombre { get; set; }
    public int Edad { get; set; }

    public Persona(string nombre, int edad)
    {
        Nombre = nombre;
        Edad = edad;
    }
}
class Program
{
    static void Main(string[] args)
    {
        Persona p1 = new Persona("Miguel Angel", 19);
        Console.WriteLine($"{p1.Nombre}, {p1.Edad}");
    }
}

File: Java/Ejercicio1.java
class Persona {   
    String nombre;
    int edad;
    Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }
    void mostrarDatos() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad);
    }
}
public class Ejercicio1 {
    public static void main(String[] args) {
        Persona p1 = new Persona("Miguel Angel", 19);
        p1.mostrarDatos();
    }
}